src/components/Dashboard/Tiles/Widgets/WidgetsMenu.tsx

```diff
@@ -63,7 +63,7 @@ export const WidgetsMenu = ({ integration, widget }: WidgetsMenuProps) => {
   const handleEditClick = () => {
     openContextModalGeneric<WidgetEditModalInnerProps>({
       modal: 'integrationOptions',
-      title: <Title order={4}>{t('descriptor.settings.title')}</Title>,
+      title: t('descriptor.settings.title'),
```

Comment:
Why? leave it as it is, this is out of scope.

Reply:
You're right, I decided to apply the boy scout rule since this caused a console error and was an easy fix.

Reply:
I think you should leave it, no one will bother to make a PR for that otherwise 🤣

Reply:
It's not causing any console errors on my side, do you have a screenshot of the error?

Reply:
It only appears when the edit dialog of a widget is opened
![image](https://github.com/ajnart/homarr/assets/31685172/5dad47dc-e21c-4c49-9867-153c2c10e0a4)

Reply:
Ah, I see where the problem lies.
unfortunately the modal we use is premade by mantine and we were not happy with the size of the title (Which just appears as normal sized text), so we added the title. But as it turns out it was already a title.
If we want to keep the right size of the title, the real fix would be to put the element in a `<section>{}</section>`
You can also add it to the handleDeleteClick just above at line 41. 

Another solution would be to edit the styling for the title, it's more correct (instead of nesting titles in titles) but more extensive.

I agree with the boy scout rule to change it, but if you can I'd rather we keep the right sizing for the title while fixing it.

Reply:
> Ah, I see where the problem lies. unfortunately the modal we use is premade by mantine and we were not happy with the size of the title (Which just appears as normal sized text), so we added the title. But as it turns out it was already a title.
> 
> I agree where you remove the title, just add this in the global.scss in styling so that the original title is the correct size:
> 
> ```css
> .mantine-Modal-title {
>   font-size: 1.375rem;
>   font-weight: 700;
> }
> ```
> 
> if you could, also remove it from the delete modal right above on line 41 too so that it's all handled.
> 
> I agree with the boy scout rule to change it, but if you can I'd rather we keep the right sizing for the title while fixing it.

Not sure if this is a correct fix as it would change for all the titles, it could create further discrepancies

Reply:
No, this would only change modal titles

Reply:
This should be okay, we might want to add some css or modal options for that

Reply:
#2019